src/Jenssegers/Mongodb/MongodbServiceProvider.php

```diff
@@ -33,5 +33,16 @@ public function register()
                 return new MongoConnector($this->app['db']);
             });
         });
+
+        // Support queue failer to record failed jobs
+        $this->app->singleton('queue.failer', function ($app) {
```

Comment:
This overwrites a Laravel component and will probably break applications if they are using a different queue backend, right?